docs/guides/biolatency.md

```diff
@@ -0,0 +1,130 @@
+
+Now, let's increase the I/O operations using the stress tool:
+
+```bash
+# Start by creating our testing namespace
+$ kubectl create ns test-biolatency
+
+# Run stress with 1 worker that will generate I/O operations
+$ kubectl run --restart=Never --image=polinux/stress stress-io -n test-biolatency -- stress --io 1
+$ kubectl wait --timeout=-1s -n test-biolatency --for=condition=ready pod/stress-io
+pod/stress-io condition met
+$ kubectl get pod -n test-biolatency -o wide
+NAME        READY   STATUS    RESTARTS   AGE   IP           NODE          NOMINATED NODE   READINESS GATES
+stress-io   1/1     Running   0          2s    10.244.1.7   worker-node   <none>           <none>
+```
+
+Using the biolatency gadget, we can generate another histogram to analyse the
+disk I/O with this load:
+
+```bash
+# Start the gadget again
+$ kubectl gadget biolatency start --node worker-node
+pCkmJ3jDtDz9yVyc
+
+# Wait again for 3 minutes
+
+$ kubectl-gadget biolatency stop pCkmJ3jDtDz9yVyc
+Tracing block device I/O... Hit Ctrl-C to end.
+
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 0        |                                        |
+         8 -> 15         : 0        |                                        |
+        16 -> 31         : 411      |                                        |
+        32 -> 63         : 310822   |****************************************|
+        64 -> 127        : 293404   |*************************************   |
+       128 -> 255        : 194881   |*************************               |
+       256 -> 511        : 96520    |************                            |
+       512 -> 1023       : 33756    |****                                    |
+      1024 -> 2047       : 4414     |                                        |
+      2048 -> 4095       : 1007     |                                        |
+      4096 -> 8191       : 1025     |                                        |
+      8192 -> 16383      : 176      |                                        |
+     16384 -> 32767      : 13       |                                        |
+     32768 -> 65535      : 7        |                                        |
+     65536 -> 131071     : 0        |                                        |
+    131072 -> 262143     : 0        |                                        |
+    262144 -> 524287     : 1        |                                        |
+    524288 -> 1048575    : 0        |                                        |
+   1048576 -> 2097151    : 1        |                                        |
+
+# Remove load
+$ kubectl delete pod/stress-io -n test-biolatency
+```
+
+The new histogram shows how the number of I/O operations increased
+significantly, passing from 1544 (Normal load) to 936438 (Stressing the I/O).
```

Comment:
The ones inside the parentheses should be lowercase IIUC from http://englishplus.com/grammar/00000123.htm.